src/Parser/MarksmanshipHunter/Modules/Features/CooldownTracker.js

```diff
@@ -15,18 +13,10 @@ class CooldownTracker extends CoreCooldownTracker {
     },
   ];
 
-  castEventSpells = [
+  static ignoredSpells = [
+    ...CooldownTracker.ignoredSpells,
```

Comment:
Can you extend these same changes to the Mistweaver Module also? Thanks.

Reply:
Wouldn't it make sense to do this in the core?

Reply:
I believe my comment was not clear. MW make use of a similar function to remove the extra cast triggers for Chi Burst and Refreshing Jade Wind, for example. So to keep it consistent just also make those changes to the MW `CooldownTracker` module. I didn't mean to extend the core boss ability clean up to Mistweavers. I agree that should be done in the core parser.